### PR TITLE
[Feature] Open the channel when clicking on an offline channel in the sidebar

### LINKED ISSUE #3289: [Feature request] Open the channel when clicking on an offline channel in the sidebar
Hello.

Currently on Twitch, when you click on an offline channel in the sidebar that have videos you've not seen, it opens the last video.

This is really annoying and I'd prefer that it redirects to the channel that I click on, and not that it directly opens the last video available. Could this be doable?

### PR DESCRIPTION
**Pull request by whitecrownclown**
Fixes #3289

### CODE REVIEW

**Comment by whitecrownclown** [comment]
@night Done, i've removed the settings for this.

File: src/modules/hide_sidebar_elements/index.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ class HideSidebarElementsModule {
             id: 'hideFeaturedChannels',
             name: 'Hide Recommended Channels',
             defaultValue: true,
-            description: 'The left sidebar is too cluttered, so you can remove recommended channels'
+            description:
```
Comment (night) [inline src/modules/hide_sidebar_elements/index.js]: do you have an autoformatter enabled? can you undo all these changes?

Comment (whitecrownclown) [reply src/modules/hide_sidebar_elements/index.js]: Yes and yes. I usually disable prettier, sorry.

**Comment by sharkykh** [comment]
Hello, can we get this merged, please? 🙏

**Pull request merged**